packages/hemera/lib/index.js

```diff
@@ -435,8 +435,8 @@ class Hemera extends EventEmitter {
    *
    * @memberOf Hemera
    */
-  static createError (name) {
-    return SuperError.subclass(name)
+  static createError (name, customConstructor) {
+    return SuperError.subclass(name, customConstructor)
```

Comment:
dont call it customConstructor because you can also pass an object. Rename it to just `options`